pythoncz/models/github.py

```diff
@@ -125,23 +136,26 @@ def _get_nodes(node, connection_name):
         connection_nodes_count = len(connection_nodes)
         connection_nodes_total_count = node[connection_name]['totalCount']
 
-        if connection_nodes_count > connection_nodes_total_count:
+        if connection_nodes_count < connection_nodes_total_count:
             warnings.warn((
```

Comment:
The docs say

> Warning messages are typically issued in situations where it is useful to alert the user of some condition in a program, where that condition (normally) doesn’t warrant raising an exception and terminating the program.

I agree in web apps this is usually typically handled by logging, but there's no logging in `pythoncz` yet. Also, nobody reads the production logs 😄 So in most cases, the warning would appear in a terminal to someone who develops the site locally. When spotted, the developer should file an issue on this repo and the problem should be solved by raising limits or pagination.